src/Extensions/Hosting.Services/Internal/ListenerBuilder.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Fabric;
+using Microsoft.ServiceFabric.Services.Communication.Runtime;
+
+namespace Microsoft.Omex.Extensions.Hosting.Services
+{
+	internal sealed class ListenerBuilder<TServiceContext> : IListenerBuilder<TServiceContext>
+		 where TServiceContext : ServiceContext
+	{
+		public ListenerBuilder(
+			string name,
```

Comment:
Validate `name`

Reply:
It's acceptable to have an empty string value I've considered to make parameter optional, and I don't want to add more validation, https://github.com/microsoft/Omex/pull/153#discussion_r390868009